src/components/ForgotPassword/forms/BottomButtonGroup.tsx

```diff
@@ -58,17 +60,26 @@ export const BottomButtonGroup: React.FC<BottomButtonGroupProps> = ({
       </Grid>
      }
       <Grid item xs={8} display="flex" justifyContent="center">
-        <Button
+        <Button className="forgot-button"
           sx={{ mx: "auto", mt: 2, width: [, 300] }}
-          color="primary"
-          variant="outlined"
+          variant="contained"
           fullWidth
           disabled={disabledContinue}
           onClick={handleContinue}
         >
           {continueButtonLabel}
         </Button>
       </Grid>
+      <Grid item xs={8} display="flex" justifyContent="center">
```

Comment:
I think we can use the button below under !hideBack , this code of yours will appear on all stepper implementation we have. Maybe just transfer this changes to the Button below instead of creating new one.